epsagon_kubernetes_subscription.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+
+## Script to attach Epsagon Role to kubernetes
+
+function usage {
+    echo "Usage: epsagon_kubernetes_subscription.sh EPSAGON_TOKEN"
+}
+
+ROLE_URL=https://raw.githubusercontent.com/epsagon/epsagon-k8s-role/master/epsagon-role.yaml
+
+function fetch_epsagon_role {
+    echo "Fetching epsagon-role.yaml"
+    if [ -f 'epsagon-role.yaml' ] ; then
+        echo "epsagon-role.yaml already exists - using that file"
+        return 0
+    fi
+    if [ `which wget` ] ; then
```

Comment:
consider dropping, as we need curl anyway in order to send the token to epsagon. Better to make them realise they need to install curl earlier

Reply:
some times (I think it will happen for windows users) wget is present and curl isn't.